### ISSUE TITLE
Select2 with 'multiple = true' breaks adding items

### ISSUE THREAD

**Issue by electronicadept** [wbraganca/yii2-dynamicform#119]
I have a dynamic form where I wanted to add a select2 (kartik/widgets/select2) with a multiple select (multiple = true). But it breaks the dynamic form when I try to add another item. The JS error I receive in the console is "Uncaught TypeError: Cannot read property 'widgetItem' of undefined" for _updateAttrID. I've tested thorougly, and if I remove the single 'multiple = true' option it works as expected.

I've contemplated using a nested dynamic form, but I'd rather not, since it's a single select box and having to click a + button, then select another item would be much slower than simply selecting multiple items.

Is there a newer version with support for the multiple select? Or does it not exist since a nested dynamic form might accommodate similar functionality?

**Comment by tigrasti** [comment]
Do you solve the problem?

**Comment by electronicadept** [comment]
No, I never was able to solve the problem. I ended up having to restructure the layout and use different controls.

**Comment by user1007017** [comment]
yii2-dynamic-form.js
```
                       //ERROR:  Uncaught TypeError: Cannot read property 'widgetItem' of undefined
                       // Replaced line:
                       // identifiers[i] = $elem.closest(widgetsOptions[i].widgetItem).index();

                       //with PATCH:
                       if(typeof widgetsOptions[i] !== 'undefined'){
                           identifiers[i] = $elem.closest(widgetsOptions[i].widgetItem).index();
                       }
```
